webextensions/manifest/optional_permissions.json

```
@@ -666,12 +726,32 @@
             }
           }
         },
+        "search": {
+          "__compat": {
+            "description": "<code>search</code>",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
+              },
+              "firefox_android": "mirror",
```

```suggestion
              "firefox": {
                "version_added": true
              },
              "firefox_android": {
                "version_added": false
              },
```
search is supported as an optional permission: https://searchfox.org/mozilla-central/rev/3c72de9280ec57dc55c24886c6334d9e340500e8/browser/components/extensions/schemas/search.json#10,14